libs/utils/src/simple_rcu.rs

```diff
@@ -179,7 +180,7 @@ impl<'a, V> RcuWriteGuard<'a, V> {
             // the watches for any that do.
             self.inner.old_cells.retain(|weak| {
                 if let Some(cell) = weak.upgrade() {
-                    watches.push(cell.watch.0.clone());
+                    watches.push(cell.watch.lock().unwrap().subscribe());
```

Comment:
Would had thought the mutex was unnecessary here, but you mentioned that this is not send without it... Weird. I am on mobile cannot confirm it now.

Reply:
I do not get an error when I remove the mutex.